specification/fist/resource-manager/Microsoft.IoTFirmwareDefense/preview/2023-02-08-preview/common.json

```diff
@@ -0,0 +1,186 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-02-08-preview",
+    "title": "fist",
+    "description": "The definitions and parameters in this swagger specification will be used to manage the IoT Firmware Defense resources."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {},
+  "definitions": {
+    "UrlToken": {
```

Comment:
>UrlToken [](http://example.com/codeflow?start=5&length=8)

If they are SAS uris, you should document such for clarity, and mark them as x-ms-secret.

Reply:
We used to mark them as x-ms-secret in our swagger, but in a previous review in azure-rest-api-specs-pr, they told us to remove it because that was only used for marking inputs as secret (body parameters), not responses. how should we proceed?

Reply:
as discussed we will leave it out for now until the documentation is clear about whether this property applies to responses as well as requests